client/my-sites/plans-features-main/index.tsx

```diff
@@ -63,7 +63,7 @@ export interface PlansFeaturesMainProps {
 	onUpgradeClick?: ( cartItemForPlan?: MinimalRequestCartProduct | null ) => void;
 	redirectToAddDomainFlow?: boolean;
 	hidePlanTypeSelector?: boolean;
-	domainName?: string;
+	paidDomainName?: string;
```

Comment:
This will break the custom domain model in several flow. Attempting a fix now!

CC - @southp @aneeshd16